lib/action_controller/serialization.rb

```diff
@@ -6,7 +6,7 @@ module Serialization
 
     include ActionController::Renderers
 
-    ADAPTER_OPTION_KEYS = [:include, :fields, :root, :adapter]
+    ADAPTER_OPTION_KEYS = [:include, :fields, :adapter]
```

Comment:
This is a pretty big change.  Prior to this, the root option *always* went to the adapter. Now it will *always* go to the serializer.  Which code path should be dead? root to serializer or root to adapter?